lib/dependencies.sh

```diff
@@ -31,14 +31,16 @@ run_if_present() {
 
 yarn_node_modules() {
   local build_dir=${1:-}
+  $NODE_VERBOSE && local opt_verbose='--verbose' || local opt_verbose=''
```

Comment:
I'm not a fan of this ternary syntax :S

Can we just do if / else? or the case option?